libcodechecker/analyze/analysis_manager.py

```diff
@@ -520,17 +293,49 @@ def handle_failure(source_analyzer, rh, action, zip_file,
                   "from analyzer output:")
         LOG.debug(str(ex))
 
-    collect_debug_data(zip_file,
-                       other_files,
-                       rh.buildaction,
-                       rh.analyzer_stdout,
-                       rh.analyzer_stderr,
-                       action.original_command,
-                       rh.analyzer_cmd,
-                       rh.analyzer_returncode,
-                       action.directory,
-                       action.target,
-                       actions_map)
+    LOG.debug("Collecting debug data")
+
+    buildactions = [{
+        'file': next(action.sources),
+        'command': action.original_command,
+        'directory': action.directory}]
+
+    for of in other_files:
+        mentioned_file = os.path.abspath(os.path.join(action.directory, of))
+        key = mentioned_file, action.target
+        mentioned_file_action = actions_map.get(key)
+        if mentioned_file_action is not None:
+            buildactions.append({
+                'file': next(mentioned_file_action.sources),
+                'command': mentioned_file_action.original_command,
+                'directory': mentioned_file_action.directory})
+        else:
+            LOG.debug("Could not find {} in build actions.".format(key))
```

Comment:
Use % formatting in logging functions and pass the % parameters as arguments

Reply:
Ok, I'll change it. But note that in these logging functions the printed strings are assembled various ways:
```.py
LOG.debug("aaa " + bbb + " ccc")
LOG.debug("aaa {} ccc", bbb)
LOG.debug("aaa %s ccc" % bbb)
LOG.debug("aaa %s ccc", bbb)
```
You can find examples on all these throughout the code base. Without project conventions or style checker tooling it is not easy to force its common usage.